docker-compose.yml

```diff
@@ -9,6 +9,7 @@ x-aliases:
       tty: true
       volumes:
         - .:/home/circleci/app
+        - .composer:/home/circleci/.composer
```

Comment:
This way switching between different versions and tested frameworks is much more fast.

Reply:
Nice! <3